### DISCUSSION TITLE
Collapsible Navigation menu

### DISCUSSION THREAD

**Discussion by Vorkosigan76** [saadeghi/daisyui#393] [Q&A] [answered]
I was wondering if someone had developed a collapsible navigation menu like this one in daisyui

https://designreset.com/cork/ltr/demo3/

you basically see the categories in the menu, when you click the categories, they will open or close

**Comment by saadeghi** [comment, marked as answer]
You can do this by putting [`collapse`](https://daisyui.com/components/collapse) inside [`menu`](https://daisyui.com/components/menu):  
https://play.tailwindcss.com/cRLwnvZg75

**Reply by Vorkosigan76** [reply]
Yes i can do this, but it doesnt look very nice and not integrated within daisyui look and feel.

but thanks

**Comment by module17** [comment]
I believe the approach with using a `collapse` inside a `menu` is not possible if they are contained within a `drawer-side`. I cannot get `collapse-content` to appear. Is this expected behaviour?
